apps/osim/workflow.py

```diff
@@ -89,6 +95,28 @@ def classify(self, instance, state=True):
             if workflow.accepts(instance):
                 return (workflow, workflow.classify(instance)) if state else workflow
 
+    def validate_classification(self, instance, workflow, state):
+        """
+        This method will evaluate if it is possible to classify the current
+        instance as a target state
+
+        Raise exception if instance lacks requirements for the target state
+
+        This method does NOT update the flaw classification
+        """
+        for fw_workflow in self.workflows:
```

Comment:
framework, I will rename the parameter of the function to target_workflow and use the framework one as workflow instead, I believe it will improve the readability